RNWCPP/src/Libraries/Components/Flyout/FlyoutProps.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ViewProps } from 'react-native';
+
+export interface IFlyoutProps extends ViewProps {
+  isLightDismissEnabled?: boolean;
+  isOpen?: boolean;
+  onDismiss?: (isOpen: boolean) => void;
+  placement?: 'Top' | 'Bottom' | 'Left' | 'Right' | 'Full';
```

Comment:
>Top [](start = 15, length = 3)

most js stuff seems to use camelCase naming so these should be lower?  top/bottom/etc   (and the parsing code in the cpp)

Reply:
I'll fix that up. Thanks!